test/mailers/previews/user_mailer_preview.rb

```diff
@@ -5,22 +5,26 @@ def abuse_report_response
     UserMailer.abuse_report(abuse_report.id)
   end
 
-  # Sends email when an archivist adds someone as a co-creator.
-  def creatorship_notification_archivist
-    second_creatorship, first_creator = creatorship_notification_data
-    UserMailer.creatorship_notification_archivist(second_creatorship.id, first_creator.id)
-  end
+  [:series, :chapter, :work].each do |creation_type|
+    # Sends email when an archivist adds someone as a co-creator.
+    define_method :"creatorship_notification_archivist_#{creation_type}" do
+      second_creatorship, first_creator = creatorship_notification_data(creation_type)
+      UserMailer.creatorship_notification_archivist(second_creatorship.id, first_creator.id)
+    end
 
-  # Sends email when a user is added as a co-creator
-  def creatorship_notification
-    second_creatorship, first_creator = creatorship_notification_data
-    UserMailer.creatorship_notification(second_creatorship.id, first_creator.id)
-  end
+    # Sends email when a user is added as an unapproved/pending co-creator
+    define_method :"creatorship_request_#{creation_type}" do
+      second_creatorship, first_creator = creatorship_notification_data(creation_type)
+      UserMailer.creatorship_request(second_creatorship.id, first_creator.id)
+    end
+
+    next if creation_type == :work
```

Comment:
Probably worth adding a comment clarifying the why of this next. Can be a simple link to https://otwarchive.atlassian.net/browse/AO3-6710

Reply:
Good point, added.